File: SuperMessenger/SuperMessenger/Models/MessageModel.cs
﻿using SuperMessenger.Models.EntityFramework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace SuperMessenger.Models
{
    public class MessageModel
    {
        public Guid Id { get; set; }
        public string Value { get; set; }
        public DateTime SendDate { get; set; }

        public SimpleUserModel User { get; set; }
        public Guid GroupId { get; set; }
    }
}

File: SuperMessenger/SuperMessenger/react-client/src/Models/MessageModel.js
export default class MessageModel{
  constructor(id, value, sendDate, groupId, user, isConfirmed ) {
    this.id = id;
    this.value = value;
    this.sendDate = sendDate;
    this.groupId = groupId;
    this.user = user;
    this.isConfirmed = isConfirmed;
  }
}